ui/components/Queue.js

```diff
@@ -166,6 +166,10 @@ const fieldComponents = {
           </Highlight>
         )
       case 'Number':
+        if (job.progress > 100) {
+          return <div className="progress-wrapper">{job.progress}</div>
```

Comment:
Instead of this early return, wouldn't it make more sense to limit the [width style](https://github.com/vcapretz/bull-board/blob/a66144a2c0a5b631233117b4eb2928f034c9c8d4/ui/components/Queue.js#L178) to 100 like: `width: ${job.progress > 100 ? 100 : job.progress}%`?

Reply:
the catch is that we also don't render the "%" (percentage sign) in the text itself, so we would need to check the style AND the text 

the goal is that the number would not indicate percentage after is higher than 100, because if that happens it may indicate other use cases for reporting progress, as stated in the issue related to this PR

Reply:
Hm, so an arbitrary progress that happens to be below 100 would automatically be considered a percentage? I’d recommend trying to structure this a bit more, or provide the user with options for what they’d like to have displayed.

Reply:
I don’t mind this change as a temporary fix, but if so I’d suggest creating an issue for providing users with more options :)

Reply:
it depends on the use case really, this is more on Bull than here on bull-board, since they provide a `.progress` method, users can call it with whatever parameters they want (as long as Bull allows), they recommend using it for percentages and specify the value should be between 0 and 100, but nothing blocks users from using this for other stuff 

https://github.com/OptimalBits/bull/blob/develop/REFERENCE.md#jobprogress

Reply:
Ok!